sdk/dotnet/Pulumi/Serialization/Attributes.Legacy.cs

```diff
@@ -0,0 +1,81 @@
+﻿// Copyright 2016-2019, Pulumi Corporation
+
+using System;
+using Google.Protobuf.WellKnownTypes;
+
+namespace Pulumi.Serialization
+{
+    /// <summary>
+    /// Attribute used by a Pulumi Cloud Provider Package to mark Resource output properties.
+    /// </summary>
+    [AttributeUsage(AttributeTargets.Property)]
+    [Obsolete("Use Pulumi.OutputAttribute instead")]
```

Comment:
frankly, i'm ok with this being:
1. fully removed.  i.e. we just move the attributes and we break code. we're in preview, that's fine with me.
2. having these be `[Obsolete("", error: true)]` so people have to move.

Reply:
I can't make it `error: true` because the `Pulumi` project itself won't compile